Exec/science/nova/_prob_params

```diff
@@ -7,11 +7,15 @@ velpert_scale       real            1.e2_rt       y
 
 velpert_amplitude   real            1.e2_rt       y
 
-velpert_height_loc  real            6.5e3_rt      y
-
 num_vortices        integer         1             y
 
 max_num_vortices    integer         10            n
 
 xloc_vortices       real            0.0_rt        n     10
 
+ih1		    integer         0             y
+
```

Comment:
both of these (ih1 and layer_h1_base) should have "n" as the last parameter -- they are not something that is set by the user